Formula/open-mesh.rb

```diff
@@ -67,6 +67,7 @@ def install
       -lOpenMeshCore
       -lOpenMeshTools
       --std=c++11
+      -Wl,-rpath,#{lib}
```

Comment:
This one's a bit concerning. What's the error you get when you don't add this?

Reply:
Hm, I think I see. `libOpenMeshTools` has linkage with an `@rpath/libOpenMeshCore.8.1.dylib`. It's a bit sad the dynamic linker isn't able to find it when the `test` executable has explicit linkage with `libOpenMeshCore`.

Reply:
yes that's what it was.
I made these patches a while ago, so any questions I have to go test them out again.

Reply:
```
% brew test open-mesh                       
==> Testing open-mesh
==> /usr/bin/clang++ test.cpp -o test -I/opt/homebrew/Cellar/open-mesh/8.1/include -L/opt/homebrew/Cellar/open-mesh/8.1/lib -lOpenMeshCore -lOpenMeshTools --std=c++11
==> ./test
Last 15 lines from Library/Logs/Homebrew/open-mesh/test.02.test:
2021-06-29 16:26:08 -0700

./test

dyld: Library not loaded: @rpath/libOpenMeshCore.8.1.dylib
  Referenced from: /opt/homebrew/opt/open-mesh/lib/libOpenMeshTools.8.1.dylib
  Reason: image not found
Error: open-mesh: failed
An exception occurred within a child process:
  BuildError: Failed executing: ./test
/opt/homebrew/Library/Homebrew/formula.rb:2218:in `block in system'
/opt/homebrew/Library/Homebrew/formula.rb:2154:in `open'
/opt/homebrew/Library/Homebrew/formula.rb:2154:in `system'
/opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/open-mesh.rb:72:in `block in <class:OpenMesh>'
/opt/homebrew/Library/Homebrew/formula.rb:2018:in `block (3 levels) in run_test'
/opt/homebrew/Library/Homebrew/utils.rb:550:in `with_env'
/opt/homebrew/Library/Homebrew/formula.rb:2017:in `block (2 levels) in run_test'
/opt/homebrew/Library/Homebrew/formula.rb:925:in `with_logging'
/opt/homebrew/Library/Homebrew/formula.rb:2016:in `block in run_test'
/opt/homebrew/Library/Homebrew/mktemp.rb:63:in `block in run'
/opt/homebrew/Library/Homebrew/mktemp.rb:63:in `chdir'
/opt/homebrew/Library/Homebrew/mktemp.rb:63:in `run'
/opt/homebrew/Library/Homebrew/formula.rb:2269:in `mktemp'
/opt/homebrew/Library/Homebrew/formula.rb:2010:in `run_test'
/opt/homebrew/Library/Homebrew/test.rb:43:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:93:in `block in timeout'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `block in catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:108:in `timeout'
/opt/homebrew/Library/Homebrew/test.rb:48:in `<main>'
```